scripts/e2e

```diff
@@ -5,6 +5,11 @@ source $(dirname $0)/lib/debug_functions
 
 cd $(dirname $0)
 
+if [[ $5 = operator ]]; then
+   test -d ../operators/go/submariner-operator || ./codegen-operator
+   # ./build-operator
```

Comment:
Delete this?

Reply:
True

Reply:
Oh no, I should un-comment, I skipped the build to test e2e once and again without waiting for the operator to be re-built.